index.html

```diff
@@ -878,14 +878,6 @@ <h2 class='informative'>Implementation Patterns</h2>
         </p>
         <ul>
           <li>Resource Identifier: the URI of the <a>LDPR</a> that was added, changed or removed.</li>
-          <li>
-            Timestamp: a timestamp marking the time of the event. The value of the timestamp
```

Comment:
@awoods: that entire section is under data that `MUST` be included. I like the new text much better.